packages/core/src/internal-components/popup/popup.element.scss

```diff
@@ -0,0 +1,267 @@
+// Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@import './../../styles/tokens/generated/index';
+@import './../../styles/mixins/mixins';
+
+:host {
+  --backdrop-background: #{$cds-alias-object-opacity-0};
+  --layered-backdrop-background: #{$cds-alias-object-opacity-0};
+  --border-color: #{$cds-alias-object-border-color};
+  --border-width: #{$cds-alias-object-border-width-100};
+  --border-radius: #{$cds-alias-object-border-radius-100};
+  --active-corner-border-radius: 0;
+  --color: #{$cds-global-typography-color-500};
+  --close-button-offset: #{$cds-global-space-1};
+  --background: #{$cds-alias-object-interaction-background};
+  --box-shadow: #{$cds-alias-object-shadow-100};
+  --min-width: 6rem;
+  --max-width: 18rem;
+  --width: fit-content;
+  --height: auto;
+  --max-height: auto;
+  --mobile-max-height: 66vh;
+  --overflow: hidden auto; // overflow x and y settings
+
+  // these CSS custom properties are read the Clarity animation
+  --animation-duration: #{$cds-global-animation-duration-primary};
+  --animation-easing: #{$cds-global-animation-easing-primary};
+
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+
+  z-index: 1000000;
```

Comment:
I'm not sure. The z-index here arguably is not needed. It's a sledgehammer that is in place here in case something goes horribly wrong for an end user.